SweetsFactory/src/app/login/login.component.ts

```diff
@@ -9,16 +9,24 @@ import { Router } from '@angular/router';
 })
 export class LoginComponent {
     showMessage = false;
+    isAdmin = false;
```

Comment:
This can also be a property on the account, not necessarily a separate variable :)